src/style.css

```diff
@@ -0,0 +1,67 @@
+.clock_container {
+  position: relative;
+  top: 100px;
```

Comment:
why did you write top and left here?

Reply:
i want that my clock was in midle of screen

Reply:
clock moved